data/json/recipes/recipe_appliance.json

```diff
@@ -96,6 +96,39 @@
       [ [ "plastic_chunk", 20 ] ]
     ]
   },
+  {
+    "result": "chest_freezer",
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "category": "CC_APPLIANCE",
+    "subcategory": "CSC_APPLIANCE_UTILITY",
+    "copy-from": "minifreezer",
+    "skills_required": [ "fabrication", 4 ],
+    "using": [ [ "welding_standard", 600 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_plasticworking", "required": false, "time_multiplier": 1.1, "learning_time_multiplier": 0.1 },
+      { "proficiency": "prof_appliance_repair", "required": false, "skill_penalty": 0 }
+    ],
+    "qualities": [
+      { "id": "HAMMER", "level": 2 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "DRILL", "level": 1 },
+      { "id": "SCREW", "level": 1 },
+      { "id": "WRENCH", "level": 1 }
+    ],
+    "components": [
+      [ [ "sheet_metal", 30 ] ],
```

Comment:
Something went horrendously wrong here. This is 180kg alone, a chest freezer is 50kg.

Reply:
As mentioned under additional context I think all refrigeration needs a wider audit as I don't think _any_ of them are currently correct.

I based surface area off of the relative increase between the household freezer and the chest freezer, rather than the surface area of the sheet metal.

I could amend this recipe to be 10 sheet metal, which would be roughly accurate to the ~4.571m^2 surface area of the chest freezer, but then it would use the same amount of sheet metal as a minifreezer and half that of a household freezer.

Given that I wasn't really sure what I was doing with this pull request (I've never submitted a pull request on github before) I wanted to Keep It Simple Stupid for myself, to make sure I knew how to make a submission before going back in later and fixing all the recipes and adding in the rest of the missing recipes, which seemed like the best option for me as a first-time contributor.

You're absolutely correct though, the recipes are pretty wrong right now, but the issue came before my PR.

Reply:
> As mentioned under additional context I think all refrigeration needs a wider audit as I don't think any of them are currently correct.

They were just added, and Guardian cited their sources in JSON comments. I don't think they're blatantly incorrect, because I know Guardian spent weeks hunting the numbers down. The weight does seem suspiciously low, but they're definitely not all incorrect.

> You're absolutely correct though, the recipes are pretty wrong right now, but the issue came before my PR.

Yeah, but you shouldn't make the issue worse by adding more to it. I don't expect you to audit all fridge recipes, but don't copy over their mistakes. Please make sure the weight at least adds up. We really desperately need to stop normalizing the violation of laws of physics in recipes...

Reply:
Reverted to draft while I resolve the recipe